docs/console/company-configuration/providers/extensions/orchestrator-generator.md

```diff
@@ -0,0 +1,245 @@
+---
+id: orchestrator-generator
+title: Orchestrator Generator
+sidebar_label: Orchestrator Generator
+---
+
+Capability: `Orchestrator Generator`
```

Comment:
how does this line appear in the page? 🤔

Reply:
removed this line, I have specified the capability name in the text